lib/components/graphs/ThresholdGraph.tsx

```diff
@@ -18,51 +18,48 @@ import {
   UpArrowIcon,
 } from '../icons/FutureThreshold';
 
+import { getTrendAfterIndex, graphThreshold, makeGraphable } from 'lib/graphs';
+
+import { GenerationDataPoint, Site } from '~/lib/types';
+import { useSiteAggregation } from '~/lib/sites';
+import { useSiteTime } from '~/lib/time';
 import {
   generationDataOverDateRange,
   getCurrentTimeGenerationIndex,
-  graphThreshold,
-  makeGraphable,
-} from 'lib/graphs';
-
-import { getTrendAfterIndex } from 'lib/utils';
-
-import { useSiteData } from 'lib/hooks';
-import useDateFormatter from '~/lib/hooks/useDateFormatter';
-import useTime from '~/lib/hooks/useTime';
-import useSiteAggregation from '~/lib/hooks/useSiteAggregation';
-import { GenerationDataPoint } from '~/lib/types';
-
-const ThresholdGraph: FC<{ siteUUIDs: string[] }> = ({ siteUUIDs }) => {
-  const { latitude, longitude, isLoading } = useSiteData(siteUUIDs[0]);
-  const { totalExpectedGeneration, totalInstalledCapacityKw } =
-    useSiteAggregation(siteUUIDs);
+} from '~/lib/generation';
+import { timeFormat, weekdayFormat } from '~/lib/format';
+import dayjs from 'dayjs';
+
+interface ThresholdGraphProps {
+  sites: Site[];
+}
+
+const ThresholdGraph: FC<ThresholdGraphProps> = ({ sites }) => {
+  const representativeSite = sites[0];
```

Comment:
For the case with multiple sites, I don't know if showing a representative site would be correct. Instead, it might be better to just show this in the user's time zone.

Reply:
can discuss more.